### PR TITLE
chore: remove playtest branch action and adjust size labels to kalium values

### PR DESCRIPTION
**Pull request by yamilmedina**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

A bit of housekeeping.

- Remove unused playtest branch creator action
- Adjust labels of size of PR to what we have on `kalium`.
- Updates kalium ref.

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by alexandreferris — APPROVED** [review]
👌🏻

**Comment by AndroidBob** [comment]
Build [2408](https://10.10.124.134/job/AR-build-pipeline/2408/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8418618)(8418618)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8418618)(8418618)-staging-compat.apk)

**Review by ohassine — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2412](https://10.10.124.134/job/AR-build-pipeline/2412/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8419665)(8419665)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8419665)(8419665)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt [hunk 1]
```diff
@@ -348,6 +348,7 @@ fun EphemeralMessageExpiredLabel(isSelfMessage: Boolean, conversationDetailsData
 @Composable
 fun MessageExpireLabel(messageContent: UIMessageContent?, timeLeft: String) {
     when (messageContent) {
+        is UIMessageContent.Location,
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt:351]: Adding this as the kalium reference was updated.

**Comment by AndroidBob** [comment]
Build [2413](https://10.10.124.134/job/AR-build-pipeline/2413/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8419950)(8419950)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8419950)(8419950)-staging-compat.apk)

**Pull request merged**